packages/components/src/components/tablist/tablist.e2e-test.ts

```diff
@@ -0,0 +1,315 @@
+    const container = stickerSheet.getWrapperContainer();
+
+    await test.step('matches screenshot of element', async () => {
+      await componentsPage.visualRegression.takeScreenshot('mdc-tablist', { element: container });
+    });
+  });
+
+  /**
+   * ADDITIONAL LOCATORS
+   *
+   * tablist - The role=tablist element within the mdc-tablist component.
+   * tabs - The role=tab element within the mdc-tablist component.
+   * arrowButtons - The mdc-button elements within the mdc-tablist component.
+   */
+  const mdcTablist = await setup({ componentsPage });
+  const tablist = mdcTablist.getByRole('tablist');
+  const tabs = mdcTablist.getByRole('tab');
+  const arrowButtons = mdcTablist.locator('mdc-button');
+
+  /**
+   * ATTRIBUTES
+   */
+  await test.step('attributes', async () => {
+    await test.step('attribute active-tab-id is set to first tab by default', async () => {
+      const activeTab = tabs.first();
+      await expect(activeTab).toHaveAttribute('active');
+      await expect(activeTab).toHaveAttribute('tabindex', '0');
+      await expect(activeTab).toHaveAttribute('aria-selected', 'true');
+    });
+
+    await test.step('attribute active-tab-id should be set to meetings-tab', async () => {
+      await componentsPage.setAttributes(mdcTablist, { 'active-tab-id': 'meetings-tab' });
+      await expect(mdcTablist).toHaveAttribute('active-tab-id', 'meetings-tab');
+      const activeTab = tabs.last();
+      await expect(activeTab).toHaveAttribute('aria-selected', 'true');
+      await expect(activeTab).toHaveAttribute('active');
+      await expect(tabs.first()).not.toHaveAttribute('active');
+      await expect(activeTab).toHaveAttribute('aria-controls', 'meetings-panel');
+    });
+
+    await test.step('attribute data-aria-label should be set to aria-label on tablist container', async () => {
+      await componentsPage.setAttributes(mdcTablist, { 'data-aria-label': 'Media types tabs' });
+      await expect(tablist).toHaveAttribute('aria-label', 'Media types tabs');
+    });
+  });
+
+  /**
+   * ACCESSIBILITY
+   */
+  await test.step('accessibility', async () => {
+    await componentsPage.accessibility.checkForA11yViolations('tablist-default');
+  });
+
+  /**
+   * INTERACTIONS
+   */
+  await test.step('interactions', async () => {
+    await test.step('resize', async () => {
+      await tabs.nth(2).click();
+
+      await test.step('resizing the viewport to 320px width should show atleast one arrow button', async () => {
+        await componentsPage.page.setViewportSize({ width: 320, height: 600 });
+        await expect(arrowButtons.first()).toBeVisible();
+      });
+
+      await test.step('resizing the viewport to 2000px width should remove both arrow buttons', async () => {
+        await componentsPage.page.setViewportSize({ width: 2000, height: 600 });
+        await expect(arrowButtons).toHaveCount(0);
+      });
+      await componentsPage.page.setViewportSize({ width: 800, height: 600 });
```

Comment:
```suggestion
      await componentsPage.page.setViewportSize({ width: 800, height: componentsPage.page.viewportSize().width });
```

If you decide to resize the viewport , then instead of a hardcoded height, keep the height of the original viewport